test/integration/apiary-reporter-test.js

```diff
@@ -202,6 +202,11 @@ describe('Apiary reporter', () => {
         assert.isObject(receivedRequest);
         assert.nestedProperty(receivedRequest, 'resultData.request');
         assert.nestedProperty(receivedRequest, 'resultData.expectedResponse');
+        // TODO
+        // "resultData.result" contains Gavel validation result.
+        // Despite Gavel validation result not having "general" key,
+        // this test still passes. This means that the result is mutated along the way.
+        // Find out where and not do that anymore.
```

Comment:
Look for `results.general` throughout Dredd. It is used at many places for storing errors related to Dredd assessing whether the test is passing or not, but not coming from Gavel.